vembrane/cli.py

```diff
@@ -0,0 +1,184 @@
+import argparse
+import ast
+import sys
+from collections import defaultdict
+from itertools import islice, chain
+from sys import stderr
+from typing import Iterator
+
+import yaml
```

Comment:
By the by: Can you move `pyyaml = "^5.3"` from the test to the normal dependencies in `pyproject.toml`? I forgot that again and have been reminded just now ^^.

Reply:
sure can do!

Reply:
It is a test dep due to a different reason, right. So, copy not move, like you did and not how I said, is correct :).